rfcs/text/0038-event-kind-asset.md

```diff
@@ -0,0 +1,160 @@
+# 0038: Extend event.kind allowed values with asset
+<!-- Leave this ID at 0000. The ECS team will assign a unique, contiguous RFC number upon merging the initial stage of this RFC. -->
+
+- Stage: **0 (strawperson)** <!-- Update to reflect target stage. See https://elastic.github.io/ecs/stages.html -->
```

Comment:
Since the scope of this change is small and the proposal is already well detailed, I propose we target stage 2 initially instead of stage 0.

By moving to stage 2, `event.kind: asset` will be added into ECS but as a `beta` allowed value. We'll still have some flexibility to make adjustments if necessary.

@taylor-swanson if you agree with targeting to stage 2, can you capture any necessary detail in the `Scope of impact` and `Concerns`?

Reply:
@ebeahan, That seems to be reasonable to me.

@SourinPaul, do we have anything specific to call out here for Scope of Impact or Concerns? I can add them to the PR once we have them.

Reply:
cc @MikePaquette in case you have any additional input here (see my comment for Sourin immediately above).

Reply:
**Scope of Impact:**
We expect to use the `asset` value within the security solution to consistently across asset management, external asset integration, and persisting related metadata. This field may be further leveraged in o11y topology work as they also are exploring ways to extend the asset.* fields within ECS.

**Concerns:**
There are no breaking changes identified. I do not have any concerns to note at this time.

@taylor-swanson, please let me know if I can elaborate anything further. Thanks for driving this.

Reply:
Thanks, @SourinPaul, I think this looks good.

I think this could wait to Stage 3 as Eric mentioned, but to add to his comment below, should we include verbiage in description for `asset` to say something on the lines of:

> This value is used by Elastic Security for asset management solutions.
>
> Usage of this value is reserved, and data ingestion pipelines must not populate event.kind with the value "asset".

Reply:
> data ingestion pipelines must not populate event.kind with the value "asset".

For `event:kind: asset` events, data pipelines (like the Azure AD user entity integration) _will be_ populating this field, right?